_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -70,7 +70,7 @@ public function activate_module( $request ) {
 			);
 		}
 
-		if ( ! Jetpack_Plan::supports( $module_slug ) ) {
+		if ( ! Jetpack_Plan::supports( $module_slug ) || ( 'search' === $module_slug && get_option( 'has_jetpack_search_product' ) ) ) {
```

Comment:
I think this logic is not working. I can't seem to activate the search module.

Reply:
Should be fixed? 🤞